<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>s21::Messages</class>
 <widget class="QDialog" name="s21::Messages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>501</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="break_pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>break</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_pushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Reading file:</string>
   </property>
  </widget>
  <widget class="QLabel" name="file_name_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
